Sources/TokamakCore/MountedViews/MountedCompositeElement.swift

```diff
@@ -30,7 +30,7 @@ class MountedCompositeElement<R: Renderer>: MountedElement<R>, Hashable {
   var mountedChildren = [MountedElement<R>]()
   let parentTarget: R.TargetType
 
-  var state = [Any]()
+  var state = [StateLocation]()
```

Comment:
I'm a bit unsure about the overhead of the new `StateLocation` type. While previously the value was stored directly (boxed as `Any` though), now there's an associated `CurrentValueSubject` object, so at least from the memory perspective I wonder if it consumes a few more bytes per state object (at least in theory). In terms of performance I'm not able to say anything until we have a proper benchmark suite. And we totally should have benchmarks sooner rather than later, I'm chipping away small bits slowly in that direction to have a proper test suite first.

Reply:
Makes sense, I'll revert it.